_docs/quickstart.md

```diff
@@ -124,25 +126,21 @@ $
 ```
 
 To get help on any `flux` subcommand or API program, the `flux help`
-command may be used. For example, to view the man page for the `flux-up(1)`
+command may be used. For example, to view the man page for the `flux-hwloc(1)`
```

Comment:
Maybe mention here that `flux help` by itself will dump a list of commonly used commands?